Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -11689,6 +11797,40 @@ end
 					},
 					Notes = "Returns true if the specified player is queued to be transferred to a World.",
 				},
+				RegisterForgeMod =
+				{
+					Params =
+					{
+						{
+							Name = "ModName",
+							Type = "string",
+						},
+						{
+							Name = "ModVersion",
+							Type = "string",
+						},
+					},
+					Notes = "Add a Forge mod name/version to the server ping list.",
```

Comment:
They'll overwrite, with the new version. What would be the preferred error handling strategy in this case, check if a mod of the name is already registered, if so change the Lua API to return a boolean false?

Related, but perhaps out of scope: would it be possible/worthwhile to have a plugin instance somehow associated with its registered mods, so they can be automatically deregistered when the plugin unloads? If this association was present, then attempting to re-register a duplicate mod could also generate a nice log message saying which plugin already claimed it. Not clear how to implement though.